### DISCUSSION TITLE
query for relational data

### DISCUSSION THREAD

**Discussion by DavidJakszta** [prisma/prisma#19936] [Q&A]
i want to find users that have their id written in a table called follows. Users have a relational field to it so i tried it through the user table like this:
```js
    const followers = await db.user.findMany({
        where: {
            followers: {
                every: {
                    followingId: fullUser?.id
                }
            }
        }
    })
```
i have two records for the userId i tried it with but i get Users that dont even have followed anybody yet as a result.
    
Schema:
```prisma
model User {
  id               String    @id @default(cuid())
  name             String?
  username         String?   @unique
  following        Follows[] @relation("following")
  followers        Follows[] @relation("follower")
  sessions         Session?
  @@map("users")
}

model Follows {
  followerId  String
  followingId String
  createdAt   DateTime @default(now())
  follower    User     @relation("following", fields: [followerId], references: [id])
  following   User     @relation("follower", fields: [followingId], references: [id])

  @@id([followerId, followingId])
  @@map("follows")
}
```

**Comment by nurul3101** [comment]
Hello @wtfmynamefits 👋 

> I want to find users that have their id written in a table called follows

Do you mean to query all records in `Follows` table where a specific user id is either in `followerId` or `followingId`?

**Reply by DavidJakszta** [reply]
no, what i want is to find all the users who have a specific followingId in their followers relational array

**Reply by DavidJakszta** [reply]
i mean yes but i want a list of users as the end result

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
